api/spec/controllers/spree/api/orders_controller_spec.rb

```diff
@@ -9,9 +9,8 @@ module Spree
     let(:variant) { create(:variant) }
     let(:line_item) { create(:line_item) }
 
-    let(:attributes) { [:number, :item_total, :display_total, :total,
-                        :state, :adjustment_total,
-                        :user_id, :created_at, :updated_at,
+    let(:attributes) { [:number, :item_total, :display_total, :total, :state,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.